### PR TITLE
Pair solved Morse code challenge

### PR DESCRIPTION
**Pull request by Omwomo**
- Solved the Morse code challenge in pair-programming session.
- Used WSL terminal to run the code.

Screenshot (853)

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @Omwomo 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 

You have submitted a project, that is a pair progrmaming activity however your partner has not made any commit, please kindly reach out to your partner so he can make contributions and submit for a re-reviewe :+1: :sparkles: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by topeogunleye — COMMENTED** [review]
Hi @GeekyHacks, @Omwomo 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have running linters. Please ensure that your linters are running before you submit for review. Kindly set up your ruby linters please using these f[iles](https://github.com/microverseinc/linters-config/tree/master/ruby)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: morse_challenge.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def decode(string)
+  morse_code = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+                 '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M',
+                 '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S',
+                 '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z' }
+  string.split.map { |letter| morse_code[letter] }.join(' ')
+end
+
+puts decode('-- -.--   -. .- -- .')
```
Comment (t-yanick) [inline morse_challenge.rb]: - [x] Great job working on this milestone. As per project requirements, you were supposed to add individual methods for:
- #decode_char method which translates Morse code symbols into letters.
**Example**: `p decode_char('.-')` or `decode_char('.-')` should return `"A"`
- #decode_word method which combines translated chars into a single word to be returned as the final output. It should reuse the `#decode_char` method
**Example**: `p decode_word('-- -.--')` or `decode_word('-- -.--')` should return `"MY"`.

At the end add the following lines of code below and run `ruby morse_challenge.rb` from the terminal
```
p decode_char('.-') or decode_char('.-')
p decode_word('-- -.--') or decode_word('-- -.--')
p decode("-- -.--   -. .- -- .") or decode("-- -.--   -. .- -- .")
p decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...") or
decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```

You should have the results below:
```
"A"
"MY"
"MY NAME"
"A BOX FULL OF RUBIES"
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @GeekyHacks and @Omwomo , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @Omwomo and @GeekyHacks  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**